templates/account.php

```diff
@@ -294,12 +294,12 @@ class="nav-tab"><?php echo esc_html( $addons_text ) ?></a>
                                                     <?php wp_nonce_field( 'delete_account' ) ?>
                                                     <a class="fs-delete-account" href="#" onclick="if (confirm('<?php

```

Comment:
@xiahengchen 

Since we are touching this, let's fix some of the semantics here too. I would love for you to:

- [ ] Create a sub template for this `if ( ! $is_paying ) : ` under `templates/account/is-paying-disconnect-button.php` and move the code there.

**templates/account/is-paying-disconnect-button.php**
```php
<?php
/**
 * @var Freemius $fs
 */
?>
<form action="<?php echo esc_attr( $fs->_get_admin_page_url( 'account' ); ?>" method="POST">
 ....
</form>
```

Do mind the usage of `esc_attr` and [other escaping functions](https://developer.wordpress.org/plugins/security/securing-output/#escaping).

In the `if` block of `templates/account.php` simply `require` (not include) that sub template file.

- [ ] Change the anchor to a button. Do add relevant CSS for it to be in the same style.

```html
<?php
// On top of the template file, calculate in advance the ID of the button.
$button_id = 'fs-disconnect-button-' . $fs->get_id();
// Also the message.
$message = $is_active_subscription ?
     fs_text_inline( ... ) :
     fs_text_inline( ... );
?>
<button class="fs-delete-account fs-button-inline" id="<?php echo esc_attr( $button_id ); ?>"> ...</button>
```

- [ ] Add inline JavaScript in the sub template file for handling the click.

```html
<script type="text/javascript">
// Wrap in a IFFE to prevent leaking global variables.
(function() {
  var disconnect_button_id = <?php echo wp_json_encode( $button_id ); ?>;
  var message = <?php echo wp_json_encode( $message ); ?>;

  jQuery(document).ready(function($) {
      $('#' + disconnect_button_id).click(function(e) {
          // Prevent the form being submitted.
          e.preventDefault();
          // Show the confirm and if yes, then submit the form.
      });
  });
})();
</script>
```

This will probably take you 30mins to 1hr to implement, but we will have a better codebase 😃 . If we follow this strategy for all PRs, then eventually our codebase will become better 👍